layer_test.go

```diff
@@ -17,6 +17,7 @@
 package libcnb_test
 
 import (
+	"io/ioutil"
```

Comment:
Was this needed?

Reply:
Yes, Idk how that happened, but the test was failing locally for me.